Database/Corrections/cataQuestFixes.lua

```diff
@@ -1754,6 +1754,9 @@ function CataQuestFixes.Load()
         [14379] = { -- Rock Lobstrock!
             [questKeys.preQuestGroup] = {14373,14374},
         },
+        [14381] = { -- Cleansing Our Crevasse
+            [questKeys.objectives] = {{{36232,nil,Questie.ICON_TYPE_MOUNT_UP},{36228},{36230},{36229}}},
```

Comment:
Now you are only copying what the database already have and change the icon for the first NPC. The last three objectives are "kill credit bunny" objectives. Meaning while they are marked as NPCs in the database, they are no real NPCs (blizzard does that every now and then for a quest). You need to find the actual NPCs to replace them.

Reply:
https://github.com/Questie/Questie/pull/6211/commits/0cdd91e9b34fbc7bb13891bd9c4191f5918dce99

Should fix these issues. The first NPC needs to have a `ICON_TYPE_MOUNT_UP` as the player needs to ride Grimfeather to do the other objectives. The bunnys have been switched with true NPC IDs.